apollo-ios/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.7
+// swift-tools-version:5.9
```

Comment:
This will require users to be on Xcode 15; which is fine as long as we're comfortable with that requirement. 15.1 should be available soon so there has been some time for adoption since it was first released. I don't know if there is a reason someone would want to stay on 14.x though?

Reply:
Yea I think its reasonable at a certain point to expect Xcode 15 especially since technically users could stay on a previous version.

Reply:
With this in mind, I imagine we would want to this update to be in a minor release that the cutoff for Xcode 15 is not in a patch release.